### PR TITLE
Add pre-commit hook for helm charts

### PR DESCRIPTION
**Pull request by miseyu**
## Summary
- Added `pre-commit hook` to check if `README.md` and `values.schema.json` for each `helm-charts` are generated automatically.

### CODE REVIEW

**Review by superbrothers — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -29,6 +31,15 @@ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
 
 You can then run `helm search repo scalar-labs` to see the Scalar charts.
 
+### Pre-commit hook
+
+If you want to automatically generate `README.md` files and `values.schema.json` with a pre-commit hook.
```
Comment (feeblefakie) [inline README.md:36]:
```suggestion
If you want to automatically generate `README.md` files and `values.schema.json` with a pre-commit hook,
```

File: README.md [hunk 2]
```diff
@@ -29,6 +31,15 @@ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
...
+Then run:
```
Comment (feeblefakie) [inline README.md:37]:
```suggestion
then run:
```

File: README.md [hunk 3]
```diff
@@ -29,6 +31,15 @@ helm repo add scalar-labs https://scalar-labs.github.io/helm-charts
...
+
+``` bash
+ln -s ../../.git-hooks/pre-commit .git/hooks/pre-commit
```
Comment (feeblefakie) [inline README.md:40]: Are you assuming you are somewhere specifically?

**Review by feeblefakie — COMMENTED** [review]
Sorry for the late review.
I left a minor question.

File: README.md (hunk 3 above)
Comment (miseyu) [reply README.md:41]: It is assumed to be run as root on the repository.
To make it clear that you are running directly under the repository, make sure to move to root.

https://github.com/scalar-labs/helm-charts/pull/72/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R40

**Comment by miseyu** [comment]
@feeblefakie 

I fixed the points that were pointed out. :bow:
https://github.com/scalar-labs/helm-charts/pull/72/commits/401a7a5c2b82de65fda4207b50f768edd3bd88b4

PTAL

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**